python/lib/sift_py/file_attachment/metadata.py

```diff
@@ -49,13 +62,16 @@ def from_pb(cls, message: VideoMetadataPb) -> Self:
             width=message.width,
             height=message.height,
             duration_seconds=message.duration_seconds,
+            timestamp=message.timestamp.ToDateTime(),  # type: ignore
```

Comment:
Just curious what the typing issue was here?

Reply:
```
lib/sift_py/file_attachment/metadata.py:65: error: "Timestamp" has no attribute "ToDateTime"  [attr-defined]
```
I think it's failing to find the attribute because this is a protobuf.

Reply:
Hm gotcha. I would really like to understand better why there are so many issues with these tools understanding protobuf imports (like in my own local setup), but I think it's alright for now, especially since this isn't even an optional member so there shouldn't be any issue.